src/App.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Todo } from './types/Todo';
+import './App.css';
+import { getTodos } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completeTodoFilter, setCompleteTodoFilter] = useState('all');
+  const [query, setQuery] = useState('');
+  const [selectedTodosId, setSelectedTodosId] = useState<number | null>(null);
```

Comment:
Todo -> one
Todos -> many
```suggestion
  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
```